atlassian/jira.py

```diff
@@ -1,6 +1,8 @@
 # coding=utf-8
 import logging
 import re
+from typing import Union
```

Comment:
does that dependency typing is built-in ?

Reply:
Typing is built into python 3, shouldn't be an issue if it's included.

Reply:
But we have still python2 clients ;(

Reply:
you should add python2 to the ci, as right now it only contains python 3. Rolled back the typing changes here: 84d5bc5  :+1:

Reply:
Main idea is remove python2 support, but time to time I receive emails ;)
That's why I have mentioned ;)